### DISCUSSION TITLE
How do I configure customize size calculator? ( height problem when I try to make it 200)

### DISCUSSION THREAD

**Discussion by congtungWonder** [MessageKit/MessageKit#1682] [Q&A]
<img width="300" alt="Screen Shot 2021-07-11 at 11 05 51" src="https://user-images.githubusercontent.com/76415753/125182432-37f85680-e238-11eb-82b8-d886a32f2936.png">
<img width="717" alt="Screen Shot 2021-07-11 at 11 08 17" src="https://user-images.githubusercontent.com/76415753/125182452-51010780-e238-11eb-9bc4-105145ef612b.png">
It displayed 200 for a while then resize to be a small rectangle same as default like image above

**Comment by maheenqaisar** [comment]
To configure a customized size calculator in Swift, you will need to use the `UIView.frame` property. This property takes four parameters: the x-coordinate, y-coordinate, width, and height of the frame. To set the height to 200, you can use the following code: `myView.frame = CGRect(x: 0, y: 0, width: myView.frame.width, height: 200)`.
